images/openshift-enterprise-pod.yml

```diff
@@ -1,15 +1,10 @@
-container_yaml:
-  go:
-    modules:
-    - module: github.com/openshift/images/pod
-      path: pod
 content:
   source:
-    dockerfile: pod/Dockerfile.rhel
+    dockerfile: build/pause/Dockerfile.rhel
```

Comment:
Two problems with this:

1. The file name is `Dockerfile.Rhel` with a capital `R`.
2. This would clone the entire kubernetes repo, and treat the top directory thereof as the working directory of the image build.  However, your dockerfile is designed to be run from within the build/pause subdirectory.  To accomplish that, you would need to set content.source.path instead and drop the full path here.

Reply:
So remove content.source.dockerfile and set content.source.path to `github.com/openshift/kubernetes/build/pause/Dcokerfile.Rhel` instead or keep content.source.dockerfile and set content.source.path to `github.com/openshift/kubernetes/build/pause`?

Reply:
I think I got it, should be fixed now. Thanks for the help!